cmds/umount/umount_linux.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
```

Comment:
I just checked in the go source tree. 

The way this worked in the beginning was that the naming directed building, so *_linux.go would only be built if GOOS was linux. The build constraints came later and are not needed AFAIK on files named for GOOS.